app/containers/Header/index.js

```diff
@@ -104,7 +104,7 @@ class Header extends React.PureComponent { // eslint-disable-line react/prefer-s
             >
               <AccountButton margin="9px 24px 9px 0" />
             </ProtectedContent>
-            <Button to="/p/happybirthday_mipt" type="header" margin="9px 0"><FormattedMessage {...messages.promo} /></Button>
+            <Button to="https://fund.mipt.ru/p/events/happybirthday_mipt" type="header" margin="9px 0"><FormattedMessage {...messages.promo} /></Button>
```

Comment:
Полный путь в Link react-router указывать необязательно.
https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md

Reply:
404 выдает с мобильной версии если полный  не указать
```suggestion
            <Button to="https://fund.mipt.ru/p/events/happybirthday_mipt" type="header" margin="9px 0"><FormattedMessage {...messages.promo} /></Button>
```

Reply:
Проблема есть, но она не так чинится. Это не стоит хардкодить. Если мы сменим домен придется урл здесь тоже менять

Reply:
Так чинится
```suggestion
          {this.props.banners.map((item, index) => (
             <Button key={index} to={item.url} type={item.type} margin="9px 0 9px 20px">{item.name}</Button>
          ))}
```